src/_slugline_vars.scss

```diff
@@ -13,3 +13,20 @@ $link-color: black;
 // Navbar variables
 $navbar-light-color: $primary;
 $navbar-brand-font-size: 2rem;
+
+// Forms
+$input-border-radius: 0;
+$input-border-width: 1px;
+$input-border-color: black;
+$input-focus-box-shadow: none;
+$input-focus-border-color: $primary;
```

Comment:
We should look into reducing the horizontal padding in text inputs here.

Reply:
Cut the padding in half, let me know what you think about it.